sources/awscli-2.patch

```diff
@@ -0,0 +1,48 @@
+diff -urN aws-cli-2.4.12-orig/awscli/customizations/wizard/app.py aws-cli-2.4.12/awscli/customizations/wizard/app.py
+--- aws-cli-2.4.12-orig/awscli/customizations/wizard/app.py	2022-01-18 22:00:18.000000000 +0000
++++ aws-cli-2.4.12/awscli/customizations/wizard/app.py	2022-01-20 22:02:20.045784837 +0000
+@@ -12,10 +12,10 @@
+ # language governing permissions and limitations under the License.
+ import json
+ import os
++from asyncio import get_event_loop
+ from collections.abc import MutableMapping
+ 
+ from prompt_toolkit.application import Application
+-from prompt_toolkit.eventloop import get_event_loop, run_until_complete
+ 
+ from awscli.customizations.wizard import core
+ from awscli.customizations.wizard.ui.style import get_default_style
+@@ -68,7 +68,7 @@
+         loop.set_exception_handler(self._handle_exception)
+         try:
+             f = self.run_async(pre_run=pre_run)
+-            run_until_complete(f)
++            loop.run_until_complete(f)
+             return f.result()
+         finally:
+             loop.set_exception_handler(previous_exc_handler)
+diff -urN aws-cli-2.4.12-orig/setup.cfg aws-cli-2.4.12/setup.cfg
+--- aws-cli-2.4.12-orig/setup.cfg	2022-01-18 22:00:18.000000000 +0000
++++ aws-cli-2.4.12/setup.cfg	2022-01-20 21:49:45.335819606 +0000
+@@ -28,14 +28,13 @@
+ python_requires = >=3.7
```

Comment:
We don't need to specify the python version for Fedora. It's consistent per release.

Reply:
Oh that is coming from the [`setup.cfg`](https://github.com/aws/aws-cli/blob/v2/setup.cfg#L28) file in the repository. In the patch file, I did not change this line and I think it is just showing up based on the width being showed in the diff. In general, I was trying to keep the patches minimal so hopefully we can do away with the patch once we the CLI source is updated to be able to pull in all of these newer versions of our dependencies.